resources/provider/migrations/postgres/22_v0.10.14.up.sql

```diff
@@ -1,38 +1,43 @@
 -- Resource: ec2.instance_statuses
-CREATE TABLE IF NOT EXISTS "aws_ec2_instance_statuses" (
```

Comment:
Should not edit migration files from previously tagged releases, this causes confusion down the line. Please revert this change.

Reply:
only formatting has changed

Reply:
would maybe discard any changes to this file